internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -272,5 +272,10 @@ func (ss *StageState) HumanString() string {
 	if transition == "" {
 		transition = empty
 	}
-	return fmt.Sprintf("  %s\t%s\t%s\n", ss.StageName, status, transition)
+	var formattedActions []string
+	for _, action := range ss.Actions {
+		formattedActions = append(formattedActions, "    "+action.Name+"\t"+action.Status+"\n")
+	}
```

Comment:
Suggestion: You could implement a `HumanString` method on the StageAction, similar to what's done on `ecs.TaskStatus`, e.g. (https://github.com/aws/copilot-cli/blob/master/internal/pkg/aws/ecs/ecs.go#L69). Then from pipeline_status, you can loop through the actions and simply call `HumanString` on them directly, similar to https://github.com/aws/copilot-cli/blob/master/internal/pkg/describe/status.go#L152-L154.

Reply:
Interesting. But describe/pipeline_status is already calling on _this_ HumanString method on the whole of the stage (https://github.com/aws/copilot-cli/blob/3d8ccdacfd6168232b8ba8efba5a5a1065064968/internal/pkg/describe/pipeline_status.go#L73). I could make a helper method within this file that each action calls within this HumanString...but is that worth it for these few lines of code that are only used here?

Reply:
right, what I'm saying is you can loop through the stages from within the main HumanString call (see linked example from previous comment). It's not a big deal, but it might make the formatting issue a little easier.

Reply:
Okay. Implemented what I think you meant--a method on StageAction, called from within the existing HumanString method in codepipeline, which is in turn called by describe/pipeline_status.